src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -347,6 +361,9 @@ public Integer invoke(File f, VirtualChannel channel) throws IOException, Interr
 
         @Override public void cleanup(FilePath workspace) throws IOException, InterruptedException {
             controlDir(workspace).deleteRecursive();
+            if (cleanupList != null) {
+                cleanupList.stream().forEach(IOUtils::closeQuietly);
```

Comment:
IMO `closeQuietly` should be deprecated and `Throwable.addSuppressed` used instead, but in practice it scarcely matters since I have never heard of an actual `close` method that actually ever throws an exception.